flink-runtime/src/main/java/org/apache/flink/runtime/executiongraph/IntermediateResultPartition.java

```diff
@@ -69,8 +71,8 @@ public ResultPartitionType getResultType() {
         return totalResult.getResultType();
     }
 
-    public List<ConsumerVertexGroup> getConsumerVertexGroups() {
-        return getEdgeManager().getConsumerVertexGroupsForPartition(partitionId);
+    public ConsumerVertexGroup getConsumerVertexGroup() {
+        return checkNotNull(getEdgeManager().getConsumerVertexGroupForPartition(partitionId));
```

Comment:
I'm wondering if there is any case that an IntermediateResultPartition has no consumers?

Reply:
There should be no such case that the moment. That's why I add `checkNotNull` here.